src/Symfony/Component/ErrorRenderer/ErrorRenderer/JsonErrorRenderer.php

```diff
@@ -40,12 +40,18 @@ public function render(FlattenException $exception): string
     {
         $debug = $this->debug && ($exception->getHeaders()['X-Debug'] ?? true);
 
+        if (404 === $exception->getStatusCode()) {
+            $message = 'Sorry, the page you are looking for could not be found.';
```

Comment:
This means you loose the original error message in debug mode which could contains some usefull details to help debug.

Reply:
Fixed now, thanks!